packages/react-composites/playwright.config.ts

```diff
@@ -9,6 +9,10 @@ const DESKTOP_VIEWPORT = {
   height: 768
 };
 
+const TEST_ROOT = './tests/browser';
+
+const buildFlavor: 'beta' | 'stable' = process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' ? 'stable' : 'beta';
```

Comment:
I hope we don't add more flavors or we may miss some update spots 👀

Reply:
Yeah, it's not super crisp how we've implemented this.

But then, even if we do add another flavor (`alpha`) it's a one-time pain. Not a routine task.